index.html

```diff
@@ -1,12 +1,256 @@
+      </article>
+      <article>
+        <img src="img/#.png" alt="#" width="1" height="1" title="#">
+        <div>415 ₽/кг</div>
+        <h4>Сливочное крем-брюле с карамельной подливкой</h4>
+      </article>
+    </section>
+
+    <section>
+      <h2>Магазин Глэйси – это онлайн и офлайн-магазин по продаже мороженого собственного проиводства на развес</h2>
+      <ul>
+
+        <li>
+          <svg></svg>
+          Все наше мороженое изготавливается на собственном производстве, с использованием современного оборудования и проверенных временем технологий.
+        </li>
+        <li>
+          <svg></svg>
+          Закупка ингредиентов производится только у проверенных фермерских хозяйств и компаний, с которыми нас связывает долговременное сотрудничество.
+        </li>
+        <li>
+          <svg></svg>
+          Для приготовления мороженого используются настоящие сливки и молоко высочайшего качества. Все дополнительные ингредиенты и добавки произведены из натурального, экологически чистого сырья.
+        </li>
+        <li>
+          <svg></svg>
+          Доставка нашего мороженного до заказчиков осуществляется в специальном термопаке, который не дает мороженому растаять в пути и позволяет сохранить превосходный вкус.
+        </li>
+      </ul>
+    </section>
+
+    <section>
+      <h2 class="visually-hidden">Новости</h2>
+      <article>
+        <h3>Новое в нашем блоге</h3>
+        <a href="#">10 способов сервировки фруктовых щербетов к столу</a>
+      </article>
+
+      <article>
+        <h3 class="visually-hidden">Подписаться на рассылку</h3>
+        <p>Подпишитесь на нашу сладкую рассылку и будьте всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)
+        </p>
+
+        <form method="post" action="https://echo.htmlacademy.ru">
+          <label>
+            <input type="email" name="#" value="#">
+          </label>
+          <button type="submit" name="#">Отправить</button>
+        </form>
+      </article>
+    </section>
+
+    <section class="container-map">
+      <h3>Как нас найти</h3>
+      <img src="#.jpg" alt="#">
+      <p>Адрес главного офиса и офлайн-магазина:<br><span>ул. Большая Конюшенная, 19/8, Санкт-Петербург</span>
+      </p>
+      <p>Для заказов по телефону:<br><a href="tel:+78124502525" aria-label="Позвонить">8 812 450-25-25</a><br>(с 10 до 20 ежедневно)
+      </p>
+      <button type="button">Форма обратной связи</button>
+    </section>
+  </main>
+  <footer>
+    <ul>
+      <li><a href="#"> <svg></svg> фб</a> </li>
+      <li><a href="#"> <svg></svg> вк</a> </li>
+      <li><a href="#"> <svg></svg> инст</a> </li>
+      <li><a href="#"> <svg></svg> тви</a> </li>
+    </ul>
+    <ul>
+      <li><a href="#"> <svg></svg>Для поставщиков </a> </li>
+      <li><a href="#">О производстве </a> </li>
+      <li><a href="#">Наши документы </a> </li>
+      <li><a href="#">Экологические стандаты </a> </li>
```

Comment:
```suggestion
      <li><a href="#">Экологические стандарты </a> </li>
```

текст удобно копировать прям из фигмы, не надо писать руками